src/python/ensembl/io/genomio/events_loader.py

```diff
@@ -0,0 +1,166 @@
+            mappings[release] = MapSession(release, event.release_date)
+
+    # Then, add the mapping, and the events for this mapping
+    for release, mapping in mappings.items():
+        print(f"Adding mapping for release {release} ({len(mapping.events)} events)")
+        if update:
+            map_session = MappingSession(new_release=mapping.release, created=mapping.release_date)
+            session.add(map_session)
+            session.flush()
+            session.refresh(map_session)
+            for event in mapping.events:
+                id_event = StableIdEvent(
+                    mapping_session_id=map_session.mapping_session_id,
+                    old_stable_id=event.from_id,
+                    new_stable_id=event.to_id,
+                    id_type="gene",
+                    old_version=1,
+                    new_version=1,
+                )
+                session.add(id_event)
+            session.commit()
+
+
+class InputSchema(argschema.ArgSchema):
+    """Input arguments expected by this script."""
+
+    # Server parameters
+    host = argschema.fields.String(
+        required=True, metadata={"description": "Host to the server with EnsEMBL databases"}
+    )
+    port = argschema.fields.Integer(required=True, metadata={"description": "Port to use"})
+    user = argschema.fields.String(required=True, metadata={"description": "User to use"})
+    password = argschema.fields.String(required=False, metadata={"description": "Password to use"})
+    database = argschema.fields.String(required=True, metadata={"description": "Database to use"})
+    input_file = argschema.fields.InputFile(
+        required=True,
+        metadata={
+            "description": (
+                "Tab input events files in the format exported by the dumper:"
+                "old_id, new_id, event_name, release, date"
+            )
+        },
+    )
+    update = argschema.fields.Boolean(
+        default=False, required=False, metadata={"description": "Set this to actually make changes to the db"}
+    )
+
+
+def make_mysql_url(host: str, user: str, database: str, port: int = 0, password: str = "") -> str:
+    """Create a mysql URL given server parameters for SQLalchemy."""
+    user_pass = user
+    host_port = host
+    if password:
+        user_pass = f"{user}:{password}"
+    if port:
+        host_port = f"{host}:{port}"
+    db_url = f"mysql://{user_pass}@{host_port}/{database}"
+    return db_url
+
+
+def main() -> None:
+    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
+
+    # Start
+    host = mod.args["host"]
+    port = int(mod.args["port"])
+    user = mod.args["user"]
+    password = mod.args.get("password")
+    database = mod.args.get("database")
+    db_url = make_mysql_url(
+        host=host,
+        port=port,
+        user=user,
+        password=password,
+        database=database,
+    )
+    dbc = DBConnection(db_url)
+
+    events = load_events(Path(mod.args.get("input_file")))
```

Comment:
If you follow my suggestion above regarding `PathLike`:
```suggestion
    events = load_events(mod.args.get("input_file"))
```